python/ray/util/sgd/v2/__init__.py

```diff
@@ -1,13 +1,15 @@
 from ray.util.sgd.v2.backends import (BackendConfig, HorovodConfig,
                                       TensorflowConfig, TorchConfig)
-from ray.util.sgd.v2.callbacks import SGDCallback
+from ray.util.sgd.v2.callbacks import SGDCallback, JsonLoggerCallback, \
+    TBXLoggerCallback
```

Comment:
If we want to leave this at the package level can we just add `callbacks` to `__all__`?

Reply:
OK made the change